<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forget_password</class>
 <widget class="QWidget" name="Forget_password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>395</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>395</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>忘记密码</string>
  </property>
  <widget class="QLabel" name="forget_username_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>200</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;用户名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="forget_email_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>115</y>
     <width>200</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;注册邮箱&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="forget_username_edit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>346</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="forget_email_edit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>115</y>
     <width>346</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="forget_submit_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>100</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;提交验证信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLabel" name="forget_verification_code_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>200</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;验证码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="forget_verification_code_edit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>346</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="forget_get_verification_btn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>115</y>
     <width>200</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;邮箱获取验证码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>获取验证码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forget_cancle_btn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>100</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;取消验证&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="forget_tooltip_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>245</y>
     <width>600</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
